index.js

```diff
@@ -498,6 +498,26 @@ const publicApi = {
     },
 
     /**
+     * Call this if you plan on loading stylus files.
+     *
+     *     Encore.enableStylusLoader();
+     *
+     * Or pass options to the loader
+     *
+     *     Encore.enableStylusLoader(function(options) {
+     *         // https://github.com/shama/stylus-loader
```

Comment:
Could you add another line with an example there so people know how to use `options` (using a real one)?

Reply:
to be honest, there isn't much that can be configured, i can not think of a good example off the top of my head. (https://github.com/shama/stylus-loader/blob/master/index.js#L33)
The primary reason, for putting this in encore is, that adding a css preprosessor that properly works with encore is a bit much boilerplate: 
```
.addLoader({
    test: /\.styl/,
    use: [
        { loader: './node_modules/extract-text-webpack-plugin/dist/loader.js',
            options: { omit: 1, remove: true } },
        { loader: 'style-loader?sourceMap' },
        { loader: 'css-loader',
            options: {
                minimize: false, sourceMap: true, importLoaders: 0
            }
        },
        { loader: 'stylus-loader'}
    ]
})
```

Reply:
Hey @mneuhaus ,

I agree that not many options are available for that loader.
Maybe : `options.import = ['~library/index.styl'];` ?
There is an example [here](https://github.com/shama/stylus-loader#using-nib-with-stylus) in which they do something like that in order to use `nib`.

Reply:
sure, sounds fine, i added it :)